state-sync/state-sync-v1/src/bootstrapper.rs

```diff
@@ -34,18 +34,15 @@ impl StateSyncBootstrapper {
         node_config: &NodeConfig,
         waypoint: Waypoint,
         event_subscription_service: EventSubscriptionService,
+        read_only_mode: bool,
     ) -> Self {
         let runtime = Builder::new_multi_thread()
-            .thread_name("state-sync")
+            .thread_name("state-sync-v1")
             .enable_all()
             .build()
             .expect("[State Sync] Failed to create runtime!");
 
-        // We notify all reconfig subscribers upon executor proxy creation
-        let mut executor_proxy = ExecutorProxy::new(storage, executor, event_subscription_service);
-        executor_proxy
-            .notify_initial_configs()
-            .expect("Failed to notify reconfig subscribers on initialization!");
```

Comment:
Is this no longer needed because it could be done by V2?  Or is it done by the multiplexer?

Reply:
It's now done by the multiplexer 😄